src/data/scripts/campaign/fleets/VayraPopularFrontManager.java

```diff
@@ -313,4 +319,44 @@ public void reportSaturationBombardmentFinished(InteractionDialogAPI dialog, Mar
             }
         }
     }
+
+
+    public void log(LoggerLogLevel logLevel, String logMessage) {
+        switch (logLevel) {
+            case FATAL: {
+                log.fatal(logMessage);
+                if (LOG_TO_CONSOLE) Console.showMessage("[FATAL] " + logMessage);
+            }
+            break;
+
+            case ERROR: {
+                log.error(logMessage);
+                if (LOG_TO_CONSOLE) Console.showMessage("[ERROR] " + logMessage);
+            }
+            break;
+
+            case WARN: {
+                log.warn(logMessage);
+                if (LOG_TO_CONSOLE) Console.showMessage("[WARN] " + logMessage);
+            }
+            break;
+
+            case DEBUG: {
+                log.debug(logMessage);
+                if (LOG_TO_CONSOLE) Console.showMessage("[DEBUG] " + logMessage);
+            }
+            break;
+
+            case INFO: {
+                log.info(logMessage);
+                if (LOG_TO_CONSOLE) Console.showMessage("[INFO] " + logMessage);
+            }
+            break;
+
+            case NONE: {
+                // it does nothing but occupying ordinal 0
+            }
+            break;
+        }
+    }
```

Comment:
this looks useful and better than before. consider merging just for this alone, even though #46 fixes this much better.

Reply:
resolving, also this is moved to MiscUtils